examples/overlays/add-remove/bpmn-visualization.js

```diff
@@ -93826,16 +93826,7 @@ var bpmnvisu = (function (exports) {
           const h = 0;
           // END bpmn-visualization CUSTOMIZATION
           if (isEdge) {
-              const pts = state.absolutePoints;
-              if (pts.length % 2 == 1) {
-                  pt = pts[Math.floor(pts.length / 2)];
-              }
-              else {
-                  const idx = pts.length / 2;
-                  const p0 = pts[idx - 1];
-                  const p1 = pts[idx];
-                  pt = new mxgraph.mxPoint(p0.x + (p1.x - p0.x) / 2, p0.y + (p1.y - p0.y) / 2);
-              }
+              pt = this.computeEdgeBounds(state);

```

Comment:
ℹ️ I didn't get that we used the larger unminified version of the lib here. IMHO we can live with it as this is a transient solution while we don't have any release with the new feature.